src/commands/dockerInstaller.ts

```diff
@@ -72,29 +72,18 @@ export class WindowsDockerInstaller extends DockerInstallerBase {
     }
 }
 
-abstract class UnixDockerInstaller extends DockerInstallerBase {
-    protected async install(fileName: string): Promise<void> {
-        const terminal = ext.terminalProvider.createTerminal(localize('vscode-docker.commands.UnixDockerInstaller.terminalTitle', 'Docker Install'));
-        const command = this.getInstallCommand(fileName);
-        terminal.sendText(command);
-        terminal.show();
-    }
-}
-
-export class MacDockerInstaller extends UnixDockerInstaller {
+export class MacDockerInstaller extends DockerInstallerBase {
     protected downloadUrl: string = 'https://aka.ms/download-docker-mac-vscode';
     protected fileExtension: string = 'dmg';
     protected installationMessage: string = localize('vscode-docker.commands.MacDockerInstaller.installationMessage', 'Opening the Docker Desktop installer...');
     protected getInstallCommand(fileName: string): string {
         return `chmod +x '${fileName}' && open '${fileName}'`;
     }
-}
 
-export class LinuxDockerInstaller extends UnixDockerInstaller {
-    protected downloadUrl: string = 'https://aka.ms/download-docker-linux-vscode';
-    protected fileExtension: string = 'sh';
-    protected installationMessage: string = localize('vscode-docker.commands.LinuxDockerInstaller.installationMessage', 'Opening the Docker installer...');
-    protected getInstallCommand(fileName: string): string {
-        return `chmod +x '${fileName}' && sh '${fileName}'`;
+    protected async install(fileName: string): Promise<void> {
+        const terminal = ext.terminalProvider.createTerminal(localize('vscode-docker.commands.UnixDockerInstaller.terminalTitle', 'Docker Install'));
```

Comment:
```suggestion
        const terminal = ext.terminalProvider.createTerminal(localize('vscode-docker.commands.MacDockerInstaller.terminalTitle', 'Docker Install'));
```

Since it moved classes